clang/include/clang/Serialization/ASTReader.h

```diff
@@ -1205,6 +1205,12 @@ class ASTReader
   /// been completed.
   std::deque<PendingDeclContextInfo> PendingDeclContextInfos;
 
+  struct PendingPreferredNameAttribute {
```

Comment:
NIT: there's nothing specific to `PreferredName` her, could we just call it `DeferredAttribute`?
(Same for functions that have it as a substring)